WalletWasabi.Gui/ViewModels/AddressViewModel.cs

```diff
@@ -110,6 +125,18 @@ public string Label
 			}
 		}
 
+		public string ExpandMenuCaption
+		{
+			get => _expandMenuCaption;
+			private set => this.RaiseAndSetIfChanged(ref _expandMenuCaption, value);
```

Comment:
This is strange. What does it do?

Reply:
It's for modifying the message of `Hide/Show QR Code` menu option